policyengine_us/parameters/gov/usda/snap/income/deductions/utility/always_standard.yaml

```
@@ -0,0 +1,116 @@
+description: The following states always use the standard utility allowance under the SNAP program.
+AK:
+  2024-10-01: false
```

I think the tests are failing because they're running on 2023 and the parameter values are unspecified. Please leave a comment indicating when you're confident the values are correct for.
```suggestion
  2023-01-01: false
```

---

I just backdated to 2020, although I only know about Colorado. Have y'all thought about adding the ability to add a default value for breakdown parameters, so we would not have to define every state, as `false`?